src/rules/base.jl

```diff
@@ -62,7 +62,37 @@
 @scalar_rule(cotd(x), -(π / oftype(x, 180)) * (1 + Ω^2))
 @scalar_rule(sech(x), -tanh(x) * Ω)
 @scalar_rule(csch(x), -coth(x) * Ω)
-@scalar_rule(hypot(x, y), (y / Ω, x / Ω))
+
+function frule(::typeof(hypot), x::Real...)
+    Ω = hypot(x...)
+    return Ω, Rule((Δ...) -> sum(Δ .* x) * inv(Ω))
```

Comment:
```suggestion
    return Ω, Rule((Δ...) -> sum(Δ .* x) / Ω)
```

Reply:
The problem is that `Zero() / x` and `One() / x` doesn't get overdubbed, so this throws errors if all `Δ`s are `Zero()`.

Reply:
Is that because only multiplications and additions are overdubbed?

Reply:
Yes, exactly